content/docs/rfcs/34/README.md

```diff
@@ -0,0 +1,172 @@
+---
+slug: 34
+title: 34/WAKU2-PEER-EXCHANGE
+name: Waku v2 Peer Exchange
+status: raw
+category: Standards Track
+tags: waku/core-protocol
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+This document specifies a simple request-response peer exchange protocol.
+Responders send information about a requested number of peers.
+The main purpose of this protocol is providing resource restricted devices with peers.
+
+**Protocol identifier**: /vac/waku/peer-exchange/2.0.0-alpha1
+
+# Background and Motivation
+
+Resource restricted devices may not feasibly be able to take part in distributed random sampling ambient peer discovery protocols such as [33/WAKU2-DISCV5](https://rfc.vac.dev/spec/33/).
```

Comment:
```suggestion
It may not be feasible on resource restricted devices to take part in distributed random sampling ambient peer discovery protocols such as [33/WAKU2-DISCV5](https://rfc.vac.dev/spec/33/).
```